latex.bzl

```diff
@@ -79,8 +79,77 @@ _latex = rule(
     implementation = _latex_impl,
 )
 
-def latex_document(name, main, srcs = [], tags = [], cmd_flags = [], format="pdf"):
+def _latex_to_svg_impl(ctx):
+    toolchain = ctx.toolchains["@bazel_latex//:latex_toolchain_type"].latexinfo
+   
+    custom_dependencies = []
+    for deps in ctx.attr.deps:
+        for file in deps.files.to_list():
+            if file.dirname not in custom_dependencies:
+                custom_dependencies.append(file.dirname)
+    custom_dependencies = ','.join(custom_dependencies)
+
+    src = ctx.attr.src
+    if LatexOutputInfo in src:
+        input_file = src[LatexOutputInfo].file
+        input_format = src[LatexOutputInfo].format
+    else:
+        fail("LatexOutputInfo provider not available in src")
+
+    flags = []
+    if "pdf" in input_format:
+        flags.append("--flag=--pdf")
+    for cmd in ctx.attr.cmd_flags:
+       flags.append("--flag="+cmd) 
+ 
+    ctx.actions.run(
+        mnemonic = "DviSvgM",
+        use_default_shell_env = True,
+        executable = ctx.executable._tool,
+        arguments = [
+            "--dep-tool=" + toolchain.kpsewhich.files.to_list()[0].path,
+            "--tool=" + toolchain.dvisvgm.files.to_list()[0].path,
+            "--env=LIBGS" + ":" + ctx.files._libgs[0].dirname + "/lib/libgs.so",
```

Comment:
I'm not sure this is the correct way to access this library file - this would definitely not work on Windows, and likely not on macOS either?

Reply:
You are right, I guess this should have had been a draft pull-request.

I forgot to even test the other platforms.

Reply:
Should be fixed now